lib/server.js

```diff
@@ -61,41 +61,40 @@ function _reqClosed(req) {
 
 
 /**
- * helper function to help verify and flatten an array of arrays.
- * takes an arguments object and an index frmo which to slice, then
- * merges that into a single array.
+ * Verify and flatten a nested array of request handlers.
+ *
  * @private
  * @function argumentsToChain
  * @throws   {TypeError}
- * @param    {Object} args  pass through of funcs from server.[method]
- * @param    {Number} start index of args at which to start working with
```

Comment:
Yeah, I'm not sure if it was used at some point or was included to support something they would do in the future, but it doesn't need to be there.